### PR TITLE
Feature/opengraph meta data

### PR DESCRIPTION
**Pull request by st3v3nhunt**
The Open Graph  data can be tested via https://developers.facebook.com/tools/debug/og/object/

### CODE REVIEW

File: app/middleware/setLocals.js [hunk 1]
```diff
@@ -1,22 +1,27 @@
 const notFound = require('../middleware/renderer').notFound;
 
+function hostNameAndProtocol(req) {
+  return `https://${req.hostname}`;
```
Comment (markysoft) [inline app/middleware/setLocals.js:4]: Should the protocol be determined from the request, rather than assuming it is always https?

**Review by markysoft — COMMENTED** [review]
Looks good, small query about assuming 'https' protocol.

File: app/middleware/setLocals.js (hunk 1 above)
Comment (st3v3nhunt) [reply app/middleware/setLocals.js:4]: Ideally yes but the SSL termination is done at the load balancer so the request coming through to the application is `http`. Which would be fine, however, non-secure requests are redirected to `https` so we get into an infinite loop or requesting pages over `http` but having them served over `https`.

**Review by markysoft — APPROVED** [review]
LGTM

**Pull request merged**